src/utils/validation.ts

```diff
@@ -61,6 +54,7 @@ async function validateInput(
         if (await emailExists(email)) {
             validationErrors.email = 'Email address already exists';
         }
+        return validationErrors;
```

Comment:
This return statement is breaking the backend for me. I don't think it's necessary as the end of the function returns the object. The if statements are just filtering by type of validation needed and adding to the validationErrors object accordingly.

Commenting them out has fixed the issue for me...